pod-operation/src/state_machine.rs

```diff
@@ -6,11 +6,13 @@ use socketioxide::extract::AckSender;
 use socketioxide::{extract::SocketRef, SocketIo};
 use tokio::sync::Mutex;
 use tracing::info;
-
 // use crate::components::signal_light::SignalLight;
+use crate::components::lim_temperature::LimTemperature;
 
 const TICK_INTERVAL: Duration = Duration::from_millis(500);
 
+const LIM_TEMP_THRESHOLD: f32 = 150.0;
```

Comment:
I think thermal cooling/propulsion may have given us a Fahrenheit value since 150°F (~65°C) seems more reasonable than beyond water boiling. Our `LimTemperature` implementation measures in Celsius, so we'll need to use the corresponding Celsius value.

Reply:
Just confirmed with @c11whaley, the max threshold is 150°C.

Reply:
death

Reply:
Calvin confirmed last night that it is 150°F ≈ 65°C.